[2022-01-06T00:31:01.650273+01:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/" (from "http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/")" at C:\xampp\htdocs\MIAGE_PrpjetWeb_S2_symfony\projet\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/\" (from \"http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/\") at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2022-01-06T00:31:01.758907+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-01-06T00:31:02.053508+01:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"73eac7"},"request_uri":"http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/_wdt/73eac7","method":"GET"} []
[2022-01-06T00:34:23.790029+01:00] cache.WARNING: Failed to save key "App%5CKernel%23configureRoutes" of type array: rename(C:\xampp\htdocs\MIAGE_PrpjetWeb_S2_symfony\projet\var\cache\dev\pools\system\CH40ItMGXa\344bcd8e8df5,C:\xampp\htdocs\MIAGE_PrpjetWeb_S2_symfony\projet\var\cache\dev\pools\system\CH40ItMGXa\8\J\jrc1F3YyD98sKrP9TpFA): Accès refusé (code: 5) {"key":"App%5CKernel%23configureRoutes","exception":"[object] (ErrorException(code: 0): rename(C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\var\\cache\\dev\\pools\\system\\CH40ItMGXa\\344bcd8e8df5,C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\var\\cache\\dev\\pools\\system\\CH40ItMGXa\\8\\J\\jrc1F3YyD98sKrP9TpFA): Accès refusé (code: 5) at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\symfony\\cache\\Traits\\FilesystemCommonTrait.php:115)","cache-adapter":"Symfony\\Component\\Cache\\Adapter\\PhpFilesAdapter"} []
[2022-01-06T00:34:23.843109+01:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/"" at C:\xampp\htdocs\MIAGE_PrpjetWeb_S2_symfony\projet\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/\" at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2022-01-06T00:34:23.897088+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-01-06T00:34:24.157009+01:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"b074e9"},"request_uri":"http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/_wdt/b074e9","method":"GET"} []
[2022-01-06T00:34:27.703637+01:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/"" at C:\xampp\htdocs\MIAGE_PrpjetWeb_S2_symfony\projet\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/\" at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2022-01-06T00:34:27.798646+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-01-06T00:34:28.184486+01:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"06d692"},"request_uri":"http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/_wdt/06d692","method":"GET"} []
[2022-01-06T00:35:25.736723+01:00] request.INFO: Matched route "app_homepage_contact". {"route":"app_homepage_contact","route_parameters":{"_route":"app_homepage_contact","_controller":"App\\Controller\\HomepageController::contact"},"request_uri":"http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/contact","method":"GET"} []
[2022-01-06T00:35:25.751276+01:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-01-06T00:35:25.866892+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-01-06T00:35:34.576354+01:00] request.INFO: Matched route "app_homepage_hello". {"route":"app_homepage_hello","route_parameters":{"_route":"app_homepage_hello","_controller":"App\\Controller\\HomepageController::hello","firstname":"j","lastname":"p"},"request_uri":"http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/hello/j/p","method":"GET"} []
[2022-01-06T00:35:34.593863+01:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-01-06T00:35:34.729639+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-01-06T00:36:22.718785+01:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/home1"" at C:\xampp\htdocs\MIAGE_PrpjetWeb_S2_symfony\projet\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/home1\" at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/home1/\". at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:74)"} []
[2022-01-06T00:36:23.020603+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-01-06T00:36:23.022940+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-01-06T00:36:23.582274+01:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f47be7"},"request_uri":"http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/_wdt/f47be7","method":"GET"} []
[2022-01-06T00:37:04.137851+01:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/home1"" at C:\xampp\htdocs\MIAGE_PrpjetWeb_S2_symfony\projet\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/home1\" at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/home1/\". at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:74)"} []
[2022-01-06T00:37:04.491388+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-01-06T00:37:04.493370+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\xampp\\htdocs\\MIAGE_PrpjetWeb_S2_symfony\\projet\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-01-06T00:37:05.066937+01:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"7eca8c"},"request_uri":"http://localhost/MIAGE_PrpjetWeb_S2_symfony/projet/public/_wdt/7eca8c","method":"GET"} []
